libcxx/include/__algorithm/fold.h

```diff
@@ -89,9 +89,7 @@ struct __fold_left_with_iter {
   }
 };
 
-inline namespace __cpo {
 inline constexpr auto fold_left_with_iter = __fold_left_with_iter();
```

Comment:
Please make sure that in your commit message for this change (so it appears in blame easily), you clearly state that the omission of the namespace was intentional and maybe provide enough information to get them to some of the discussion that's happened here or elsewhere.

Reply:
I've updated the description in this PR, which I _think_ becomes the main message for a squash-and-merge.